### DISCUSSION TITLE
求教硕士论文封面相关问题

### DISCUSSION THREAD

**Discussion by HuTing0809** [note286/xduts#190] [Q&A] [answered]
导师要求两个指导教师必须分为两行来写，请问应该如何实现？

**Comment by note286** [comment, marked as answer]
请阅读手册5.21.1节。

**Reply by HuTing0809** [reply]
我试了一下加中文逗号的话指导教师一栏是只有一行，是张三，李四  教授，副教授的形式。
导师要求的指导教师是一行写张三 教授，第二行写李四 副教授的形式。
请问如果要实现导师说的功能是不是需要在cls文件中修改。

**Reply by note286** [reply]
@HuTing0809 你也强调了你使用了中文逗号，你为什么不试一下英文逗号呢？

**Reply by HuTing0809** [reply]
试过英文逗号，会报如图所示的错误。
![报错](https://github.com/note286/xduts/assets/74964483/0c320cba-ea86-45a0-b0e3-9e39b7ab89dc)

**Reply by sikouhjw** [reply]
试试
```tex
info/supervisor = {a, b}
```

**Reply by note286** [reply]
`{张三,李四}`，另外给个建议，不要让人引导你，最好一开始就尽可能的清晰地描述你的问题和你的尝试。

**Reply by HuTing0809** [reply]
@sikouhjw 谢谢！已解决！

**Reply by HuTing0809** [reply]
@note286 谢谢~已解决
